app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def avatar_url(user)
+    gravatar_id = Digest::MD5::hexdigest(user.email).downcase
+    "http://gravatar.com/avatar/#{gravatar_id}.png?s=80"
+  end
```

Comment:
So I couldn't really think of a way to test that other than put the class "profile-image-gravatar" onto the callback and test for that. That is located in the admin_is_logged_in_spec.rb